libsolidity/analysis/TypeChecker.cpp

```diff
@@ -1489,6 +1489,23 @@ bool TypeChecker::visit(FunctionCall const& _functionCall)
 		}
 	}
 
+	if (auto memberAccess = dynamic_cast<MemberAccess const*>(&_functionCall.expression()))
```

Comment:
Makes sense, I started thinking about putting it in the function call at first and never questioned that assumption.